routes/user.js

```diff
@@ -12,8 +12,38 @@ var crypto = require('crypto')
 var nodemailer = require('nodemailer')
 var config = require('config')
 
+// router.get('/profile', function(req, res){
```

Comment:
is profile not enabled in this commit? that's fine, but if its irrelevant to the commit, perhaps it could be cleaned up on the other fronts?